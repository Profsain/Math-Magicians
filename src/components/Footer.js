import React from 'react';
import '../styles/Footer.css'

export const Footer = () => {
  return (
    <footer className='Footer'>
      <p>Developer: Husseini Mudi @Microverse Inc</p>
    </footer>
  )
}
