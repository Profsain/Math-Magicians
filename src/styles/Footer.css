.Footer {
  padding: 8px;
  text-align: center;
  background-color: #1b1b1b;
  color: #fff;
}
