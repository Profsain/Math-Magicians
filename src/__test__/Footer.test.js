import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

it('renders component correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
