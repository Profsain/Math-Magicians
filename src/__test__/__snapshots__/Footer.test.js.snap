// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component correctly 1`] = `
<footer
  className="Footer"
>
  <p>
    Developer: Husseini Mudi @Microverse Inc
  </p>
</footer>
`;
